src/Task/Testing/PHPUnit.php

```diff
@@ -189,7 +191,7 @@ public function file($file)
      */
     public function dir($dir)
     {
-        return $this->dir($dir);
+        return $this->execOneCommandDir($dir);
```

Comment:
Couldn't we just completely delete the `dir` function? It should do the right thing then. It would be helpful if this was confirmed with a unit test.